datasets/ONS-UK-Total-trade/main.py

```diff
@@ -44,69 +42,45 @@ def date_time (date):
+
+df = pd.concat(tidied_sheets, sort = True).fillna('')
 
 #Post Processing 
-df = trace.combine_and_trace(datasetTitle, "combined_dataframe")
 df.rename(columns={'OBS' : 'Value','DATAMARKER' : 'Marker'}, inplace=True)
-#df['Value'] = df['Value'].astype(float)
-df['Value'] = pd.to_numeric(df.Value, errors='coerce')
-df['Marker'].replace('N/A', 'not-collated', inplace=True)
-trace.Marker("replacing N/A with not-collated")
-df['Flow'] = df['Flow'].map(lambda s: s.lower().strip())
-df["Country"] = df["Country"].map(lambda x: pathify(x).upper())
-df['Trade Type'] = df['Trade Type'].apply(lambda x: 'total' if 'Total Trade' in x else 
+df['Trade Type'] = df['Trade Type'].apply(lambda x: 'all' if 'Total Trade' in x else 
                                       ('goods' if 'Trade in Goods' in x else 
                                        ('services' if 'Trade in Services' in x else x)))
-trace.Trade_Type("Replacing Trade Type output to one of; total, goods, services if the title contains the one of those words.")
 df['Period'] = df['Period'].astype(str).replace('\.0', '', regex=True)
-trace.Period('Formatingyear values into year/{yr} (year/2020) and quarter values into quarter/{qtr} (quarter/Q1)')
 df['Period'] =  df["Period"].apply(date_time)
-df = df[['Period', 'Country', 'Flow', 'Trade Type', 'Value', 'Marker']]
-
-#additional scraper info needed
-scraper.dataset.family = 'trade'
-add_to_des = """
-These tables have been produced to provide an aggregated quarterly goods and services estimate and combines the most recent estimates for goods and services split by country.
-Data for goods and services is consistent for annual whole world totals and quarters (from Q1 2016) with the trade data published in the Quarterly National Accounts, Quarterly Sector Accounts and Quarterly Balance of Payments on 30th September 2020.
-These data are our best estimate of these bilateral UK trade flows. Users should note that alternative estimates are available, in some cases, via the statistical agencies for bilateral countries or through central databases such as:
-UN Comtrade.
-Some data for countries have been marked with N/A. This is because Trade in Goods do not collate data from these countries, therefore only Trade in Services is reflected within total trade for these countries
-The data within these tables are also consistent with the below releases:
-For Trade in Goods the data is consistent with UK Trade: August 2020 publication on 9th October 2020
-For Trade in Services the data is consistent with UK Trade in services by partner country: April to June 2020 publication on 4th November 2020
-"""
-scraper.dataset.description = scraper.dataset.description + add_to_des
-
-cubes.add_cube(scraper, df.drop_duplicates(), "ons-uk-total-trade")
-cubes.output_all()
-trace.render("spec_v1.html")
-
-
-df.dtypes
+df = df[['Period', 'Country', 'Flow', 'Trade Type', 'Marker', 'Value']]
+
+df['Flow'] = df['Flow'].map(lambda x: 'export'  if 'Exports' in x else 'import')
+
```

Comment:
Updated description needs to be added as per the latest publication. You could make use of landing page and "Index" tab in the spread sheet.

metadata.dataset.description = f"""add the latest description here"""

Reply:
Have updated all changes as requested.